docs/source/requirements_docs.txt

```diff
@@ -1,6 +1,8 @@
+dill
+ipython
+ipykernel
 nbsphinx
+python-dotenv[cli]
 sphinx_copybutton
 sphinx_rtd_theme
-python-dotenv[cli]
```

Comment:
The dotenv is used for loading env variables in notebooks. We have  not found any other way to bulk load env vars in NB